README.md

```diff
@@ -55,7 +72,10 @@ Describe the importance of using object-oriented programming.
 
 Your Answer:
 ```
-// your answer here
+object-oriented is important for organizing data collections.
+Using object-oriented is perfect for keeping track of info such as name, age, etc
+and when you need to add a change to a specific person,
+you can do so with a new instance of the already created Object Literal set
 ```
```

Comment:
You right in a general sense, but I would make sure to include mentions to encapsulation, business data and logic. like:
```
OOP forces you to encapsulate your code: break processes up into specific sub-processes that all interact with each other. Object-oriented code is often easier to read and easier to import without breaking existing code.
```